aa-sdk/core/src/actions/smartAccount/internal/waitForUserOperation.ts

```diff
@@ -0,0 +1,56 @@
+import type { Chain, Client, Hex, Transport } from "viem";
+import type {WaitForUserOperationTxParameters} from "../types";
+import {isBaseSmartAccountClient} from "../../../client/isSmartAccountClient";
+import {IncompatibleClientError} from "../../../errors/client";
+import {Logger} from "../../../logger";
+import {FailedToFindTransactionError} from "../../../errors/transaction";
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.29.0/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../../../errors/transaction"
<!-- __reviewdog__:ChA3NWUwZDQ5YWIzY2VlNDU5EgZyZGpzb24= -->